src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -231,6 +236,48 @@ class EthService(
     maybeTxResponse.map(txResponse => Right(GetTransactionByHashResponse(txResponse)))
   }
 
+  def getTransactionReceipt(req: GetTransactionReceiptRequest): ServiceResponse[GetTransactionReceiptResponse] = Future {
+    val result: Option[TransactionReceiptResponse] = for {
+      TransactionLocation(blockHash, txIndex) <- blockchain.getTransactionLocation(req.txHash)
+      Block(header, body) <- blockchain.getBlockByHash(blockHash)
+      stx <- body.transactionList.lift(txIndex)
+      receipts <- blockchain.getReceiptsByHash(blockHash)
+      receipt: Receipt <- receipts.lift(txIndex)
+    } yield {
+
+      //transaction with contract creation 0xb7b8cc9154896b25839ede4cd0c2ad193adf06489fdd9c0a9dfce05620c04ec1
```

Comment:
Minor detail: What does `0xb7b8cc9154896b25839ede4cd0c2ad193adf06489fdd9c0a9dfce05620c04ec1` represent? From the tests it seems to be the hash of a tx, maybe we should remove this comment?